app/database/group_attempt_store_compute_all_group_attempts.go

```diff
@@ -188,38 +185,25 @@ func (s *GroupAttemptStore) ComputeAllGroupAttempts() (err error) {
 }
 
 func (s *GroupAttemptStore) collectItemsToUnlock(groupItemsToUnlock map[groupItemPair]bool) {
-	// Unlock items depending on has_unlocked_items
+	// Unlock items according to items_unlocking_rules
```

Comment:
Not sure it still makes sense to collect the items separately from updating them (probably much more expensive)... but again we'll change this later with the update of the propagation.